diff --git a/robotiq_description/CMakeLists.txt b/robotiq_description/CMakeLists.txt
index e908b35..0047aae 100644
--- a/robotiq_description/CMakeLists.txt
+++ b/robotiq_description/CMakeLists.txt
@@ -5,9 +5,6 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic)
 endif()
 
-# find dependencies
-find_package(ament_cmake REQUIRED)
-
 install(
   DIRECTORY
     config
@@ -18,5 +15,3 @@ install(
   DESTINATION
     share/${PROJECT_NAME}
 )
-
-ament_package()
